src/AudioManagerImpl.js

```diff
@@ -1,11 +1,159 @@
 // @flow
 
-import { Midi, Panner, Sampler } from 'tone';
+import { Midi, Panner, Sampler} from 'tone';
 import CharacterState from './CharacterState';
 import type {IntlShape} from 'react-intl';
 import {AudioManager} from './types';
 import SceneDimensions from './SceneDimensions';
 
+const SamplerDefs = {
+    // The percussion instruments we use actually don't vary their pitch, so we use the same sample at different
+    // pitches so that we can scale relative to the octave without ending up with wildy different tempos.
+    left45: {
+        urls: {
+            "C0": "C6.wav",
+            "C1": "C6.wav",
+            "C2": "C6.wav",
+            "C3": "C6.wav",
+            "C4": "C6.wav",
+            "C5": "C6.wav",
+            "C6": "C6.wav"
+        },
+        baseUrl: "/audio/left/45/"
+    },
+    left90: {
+        urls: {
+            "C0": "C6.wav",
+            "C1": "C6.wav",
+            "C2": "C6.wav",
+            "C3": "C6.wav",
+            "C4": "C6.wav",
+            "C5": "C6.wav",
+            "C6": "C6.wav"
+        },
+        baseUrl: "/audio/left/90/"
+    },
+    left180: {
+        urls: {
+            "C0": "C6.wav",
+            "C1": "C6.wav",
+            "C2": "C6.wav",
+            "C3": "C6.wav",
+            "C4": "C6.wav",
+            "C5": "C6.wav",
+            "C6": "C6.wav"
+        },
+        baseUrl: "/audio/left/180/"
+    },
+    right45: {
+        urls: {
+            "C0": "C6.wav",
+            "C1": "C6.wav",
+            "C2": "C6.wav",
+            "C3": "C6.wav",
+            "C4": "C6.wav",
+            "C5": "C6.wav",
+            "C6": "C6.wav"
+        },
+        baseUrl: "/audio/right/45/"
+    },
+    right90: {
+        urls: {
+            "C0": "C6.wav",
+            "C1": "C6.wav",
+            "C2": "C6.wav",
+            "C3": "C6.wav",
+            "C4": "C6.wav",
+            "C5": "C6.wav",
+            "C6": "C6.wav"
+        },
+        baseUrl: "/audio/right/90/"
+    },
+    right180: {
+        urls: {
+            "C0": "C6.wav",
+            "C1": "C6.wav",
+            "C2": "C6.wav",
+            "C3": "C6.wav",
+            "C4": "C6.wav",
+            "C5": "C6.wav",
+            "C6": "C6.wav"
+        },
+        baseUrl: "/audio/right/180/"
+    },
+    backward1: {
+        urls: {
+            "C0": "C0.wav",
+            "C1": "C1.wav",
+            "C2": "C2.wav",
+            "C3": "C3.wav",
+            "C4": "C4.wav",
+            "C5": "C5.wav",
+            "C6": "C6.wav"
+        },
+        baseUrl: "/audio/backward/1/"
+    },
+    backward2: {
+        urls: {
+            "C1": "C1.wav",
```

Comment:
`backward2` does not have a "C0" entry. Is this an issue?

Reply:
I couldn't make the instrument I used play anything lower than C1.

Reply:
I'll try to remember how I managed it for forwards, which uses the same patch.

Reply:
OK, I had to remind myself of how I did this.  The arpeggiator built into the CS-80 (emulator) can easily create the pattern used for the forward sounds, you set the arpeggiator to "down", hold C1, and it alternates C1 and C0 (an octave lower).   Ideally you would set the arpeggiator to "up" and trigger C0, but since the CS-80 doesn't respond to anything lower than C1, I had to get creative.

I ended up using the pitch adjusting ribbon control on the CS-80, setting it to a range of 24 steps, which meant that it ranged from an octave lower (hard left) to an octave higher (hard right) than the played note.  Using that, I could hold the ribbon control hard left, hit C1 and it played C0 and then C1 as desired.

The patch gets a bit "crunchy" on the low end, but the sound is still audible, and we now have a complete set.